sdk/python/runtime/discovery.go

```diff
@@ -111,6 +118,10 @@ func (d *Discovery) UserConfig() *UserConfig {
 	return &d.Config.Tool.Dagger
 }
 
+func (d *Discovery) UvConfig() *UvConfig {
```

Comment:
Missing description:
```suggestion
// UvConfig is the configuration the user can set in pyproject.toml, under
// the "tool.uv" table.
func (d *Discovery) UvConfig() *UvConfig {
```